content/java/concepts/math-methods/terms/decrementExact/decrementExact.md

```diff
@@ -0,0 +1,49 @@
+---
+Title: '.decrementExact()'
+Description: 'Returns the argument decremented by one, throwing an exception if the result overflows the datatype.'
+Subjects:
+- 'Computer Science'
+  Tags:
```

Comment:
```suggestion
Tags:
```

Reply:
@Dusch4593 , we need the line gap between the subject, and tags. else the content will overlap.

<img width="1228" alt="Screenshot 2022-10-21 at 4 22 29 PM" src="https://user-images.githubusercontent.com/6908935/197179790-8581f597-ef82-49a1-aba7-fce317cfd442.png">
 

I am attaching [pictures.](URL)

<img width="1228" alt="Screenshot 2022-10-21 at 4 22 55 PM" src="https://user-images.githubusercontent.com/6908935/197179770-ef9d2bdb-85c4-49f9-a42f-3ae4fbec010f.png">

Reply:
Let me know your thoughts.